data/import/common/common/marketplace/merchant_product_approval_status_default.csv

```diff
@@ -1,2 +1,2 @@
 merchant_reference,approval_status
-MER000002,approved
+MER000002,draft
```

Comment:
Where did this side effect come from?

Reply:
https://github.com/spryker/suite-nonsplit/pull/6685/files